src/HomePage.js

```diff
@@ -1,12 +1,56 @@
 import React from 'react';
+import { Card } from 'react-bootstrap';
+import { Accordion } from 'react-bootstrap';
+import { Navbar } from 'react-bootstrap';
+import { Nav } from 'react-bootstrap';
+import { Container } from 'react-bootstrap';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 class HomePage extends React.Component {
+  // Un-comment this for week 4 content 
+  // constructor(props){
+  //   super(props);
+
+  //   this.state = {}
+  // }
   
   render() {
     return (
       <div>
-        {/* Complete your training in here*/}
-        <div>Welcome to VEEP Website!</div>
+        <Navbar bg="success" variant="dark">
+          <Container>
+          <Navbar.Brand href="#home">UofT VEEP</Navbar.Brand>
+          <Nav className="me-auto">
+            <Nav.Link href="#home">Home</Nav.Link>
+            <Nav.Link href="#features">Projects</Nav.Link>
+            <Nav.Link href="#pricing">About Us</Nav.Link>
```

Comment:
Even though we don't have pages to jump to right now. It's always good to change the href into corresponding name